README.md

```diff
@@ -1 +1,331 @@
+
+<!--
+Example command:
+sh
+  open live server
+
+--->
+
+## 👥 Authors
+
+👤 *Jonathan Kiorono*
+
+- GitHub: (https://github.com/Jonathan-kiprono)
+- LinkedIn: [LinkedIn][(https://www.linkedin.com/in/jonathan-kiprono-8858a9193/overlay/contact-info/)]
+
+## 🔭 Future Features
+
+- [ ] *[Add more forms]*
+- [ ] *[Add more pages with information]*
+- [ ] *[Add some Javascript]*
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## ⭐️ Show your support
+
+If you like this project feel free to share with your friends.
+
+## 🙏 Acknowledgments
+
+I would like to thank Microverse for this amazing exercise.
+
+## 📝 License
+
+This project is [MIT](https://github.com/ntimanu/portfolio-project/blob/htmlcssbranch/LICENCE) licensed.
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+# 📖 [Portfolio: setup and mobile first]
+
+*[Portfolio: setup and mobile first]* is a simple project that show a page of a portfolio that discribe the work, experience and contacts of the portfolio owner.
+
+## 🛠 Built With
+
+### Tech Stack
+
+Tech Stack
+HTML
+CSS
+JS
+Git/GitHub
+node.js
+Boostrap 5
+
+### Key Features
+
+- *[A webpage that gives you all the information of the owner]*
+
+## 🚀 Live Demo
+
+- [Comming Soon]
+
+## 💻 Getting Started
+
+When you click on the page, the browser shows you the information of the owner.
+
+### Prerequisites
+
+Clone this repository to your desired folder and open html file.
+
+### Setup
+
+Clone this repository to your desired folder:
+Example commands:
+
+sh
+  cd portfolio-project
+  git clone https://github.com/ntimanu/portfolio-project
+
+
+### Install
+
+Install this project with:
+
+<!--
+Example command:
+sh
+  cd portfolio
+  npm install
+
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+sh
+  open live server
+
+--->
+
+## 👥 Authors
+
+👤 *Jonathan Kiprono*
+
+- GitHub: (https://github.com/Jonathan-kiprono)
+- LinkedIn:[https://www.linkedin.com/in/jonathan-kiprono-8858a9193/overlay/contact-info/]
+
+## 🔭 Future Features
+
+- [ ] *[Add more forms]*
+- [ ] *[Add more pages with information]*
+- [ ] *[Add some Javascript]*
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## ⭐️ Show your support
+
+If you like this project feel free to share with your friends.
+
+## 🙏 Acknowledgments
+
+I would like to thank Microverse for this amazing exercise.
+
+## 📝 License
+
+This project is [MIT] licensed.
```

Comment:
- [ ] From your Repo and PR, I see that this project is about `Hello Microverse` but the contents are all about `portfolio project`. This is will make readers confused and even think that your work is just copy-paste activity. Kindly edit appropriately as the professional that you are 😉 🖖